crowbar_framework/app/models/deployer_service.rb

```diff
@@ -181,6 +162,27 @@ def transition(inst, name, state)
       roles << node.run_list_to_roles
       roles.flatten!
 
+      # Allocate required addresses
```

Comment:
Also: I'm wondering whether this should move to the transition method of the network service? (since `delete` / deallocating IP address is handled there)